workflow/util/util.go

```diff
@@ -341,7 +342,7 @@ func SuspendWorkflow(wfIf v1alpha1.WorkflowInterface, workflowName string) error
 
 // ResumeWorkflow resumes a workflow by setting spec.suspend to nil and any suspended nodes to Successful.
 // Retries conflict errors
-func ResumeWorkflow(wfIf v1alpha1.WorkflowInterface, workflowName string, nodeFieldSelector string) error {
+func ResumeWorkflow(wfIf v1alpha1.WorkflowInterface, workflowName string, nodeFieldSelector string, repo sqldb.OffloadNodeStatusRepo) error {
```

Comment:
Having a utility function where some of the parameters are services stinks. Ideally we should have a `resumeService`, but not today. 
I feel like repo should be the second parameter?